packages/launcher/electron/main.ts

```diff
@@ -60,3 +61,22 @@ app.on("activate", () => {
     createWindow();
   }
 });
+
+ipcMain.on("request-tags", (event) => {
+  https
+    .get("https://registry.hub.docker.com/v2/repositories/wpilib/axon/tags", (res) => {
+      res.on("data", (d: Buffer) => {
+        const data = JSON.parse(d.toString()).results.map((element: { name: string }) => {
+          return element.name;
+        });
+        event.reply("axon-tags", data);
+      });
+    })
+    .on("error", (e) => {
+      console.error(e);
+    });
+});
+
+ipcMain.on("request-version", (event) => {
+  event.reply("launcher-version", process.env.npm_package_version);
```

Comment:
Use synchronous syntax because this is not blocking

```suggestion
  event.returnValue = process.env.npm_package_version;
```